spec/features/authentication/03_sign_out_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+require 'launchy'
```

Comment:
Remove this line, you do not need to require `launchy` anywhere in a Rails app as long as it is in your Gemfile.